test/components/StatefulButton.test.tsx

```diff
@@ -5,15 +5,33 @@ import StatefulButton, { StatefulButtonProps } from '../../src/Button/StatefulBu
 import clone = require('lodash/clone');
 
 let consoleError: jest.SpyInstance<{}>;
+const consoleWarn = console.warn;
+
+const loadingSpinnerWarning =
+  `LoadingSpinner's parent node style should have "position: relative" or "position: absolute"`;
 
 beforeAll(() => {
   consoleError = jest.spyOn(console, 'error');
+  console.warn = jest.fn((...message) => {
```

Comment:
an alternative to this would've been to set `style: 'relative'` in JSX in the `Button` implementation, but this seemed less breaking.